pyradigm/pyradigm.py

```diff
@@ -415,11 +415,8 @@ def get_subset(self, subset_ids):
             return MLDataset()
 
     def __contains__(self, item):
-        "Boolean test of membership of a sample in the dataset."
-        if item in self.keys:
-            return True
-        else:
-            return False
+        """Boolean test of membership of a sample in the dataset."""
+        return item in self.keys
```

Comment:
Hey Scott, thanks for the PR. For a simple test, the suggested code is certainly more compact. However, I planned to add more functionality for [non-]membership, in case of multiple items or a more complex test (e.g. missing data). I should perhaps add a note or TODO here to reflect that.

Reply:
What more complex test would you add? Isn't missing data not contained in the dataset?

Reply:
For example, when the missing data is partial, or data is not of certain type across all subjects. I'm still thinking through other use cases, though. But thanks for the suggestion to compactify it.